terraform/deployments/cluster-infrastructure/external_secrets_iam.tf

```diff
@@ -0,0 +1,50 @@
+# IAM role and policy to enable the k8s external-secrets operator to talk to
+# AWS APIs to access and manage SecretsManager secrets.
+#
+# The k8s side of the autoscaler is in
+# ../cluster-services/external_secrets.tf.
+
+locals {
+  external_secrets_service_account_namespace = "secrets"
```

Comment:
So there's a question here around which namespaces we should have and what resources go where. A specific namespace for external-secrets feels a bit too granular to me, but I also wouldn't want it to go in `default` unless required (which I don't think it is). I'd suggest having a `cluster-services` namespace for all supporting services like secrets, DNS, etc, with the possible exception of monitoring and logging, as there will be a lot of resources there so we might want those in a `monitoring` namespace just to keep things clean.

This is up for grabs though, so comments gratefully received (@sengi ?)

Reply:
I'm easy on this. `cluster-services` and maybe `monitoring` SGTM. I'd also be ok with separate ones (though if we were to go for that I'd mildly prefer `external-secrets` as the name).

Reply:
Thanks, I've changed this to `cluster-services`.